vtex-apps/product-list
# Replaces loading spinner by skeleton

#### What problem is this solving?

This PR adds a preview skeleton to `product-list` in order to replace the current loading spinner on `checkout-cart`.



#### How should this be manually tested?

- Go to [Workspace](https://cartske--vtexgame1.myvtex.com/cart)
- Open your browser dev tools and change from 'No throttling' to 'Slow 3G' (optional)
- Verify if the skeleton is showed properly


#### Checklist/Reminders

- [x] [Linked this PR to a Clubhouse story (if applicable).](https://app.clubhouse.io/vtex/story/24408/aplicar-o-skeleton-no-carregamento-do-carrinho)

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/12574426/67590113-85fdbb80-f730-11e9-9890-9f009d2e1eb5.png)


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️| New feature 
_ | Breaking change 
_ | Technical improvements
